.github/.codecov.yml

```diff
@@ -1,3 +1,11 @@
 coverage:
   round: up
-  precision: 2
+  precision: 1
+  status:
+    project:
+      default:
+        threshold: 1%
```

Comment:
@jeremyliweishih good point! Let's try it out. I haven't been able to find docs for `threshold` other than [the link you found](https://docs.codecov.io/docs/common-recipe-list#set-status-checks-to-block-coverage).

I saw someone [on their forum used this](https://community.codecov.io/t/threshold-confusion/88/10) and didn't complain about it breaking, beyond some sort of caching bug which is hopefully now irrelevant.